definitions/npm/styled-components_v3.x.x/flow_v0.57.x-/styled-components_v3.x.x.js

```diff
@@ -343,20 +343,19 @@ declare module 'styled-components' {
   declare type ReactComponentStyled<Props>                                = $npm$styledComponents$ReactComponentStyled<Props, ComponentListKeys>;
   declare type ReactComponentStyledTaggedTemplateLiteral<Props>           = $npm$styledComponents$ReactComponentStyledTaggedTemplateLiteralWithComponent<Props, ComponentListKeys>;
 
-  declare module.exports: {
-    $call: $npm$styledComponents$Call<ComponentListKeys>,
-
-    injectGlobal: TaggedTemplateLiteral<void>,
-    css: TaggedTemplateLiteral<Array<Interpolation>>,
-    keyframes: TaggedTemplateLiteral<string>,
-    withTheme: $npm$styledComponents$WithTheme,
-    ServerStyleSheet: typeof Npm$StyledComponents$ServerStyleSheet,
-    StyleSheetManager: typeof Npm$StyledComponents$StyleSheetManager,
-    ThemeProvider: typeof Npm$StyledComponents$ThemeProvider,
-
+  declare export var $call: $npm$styledComponents$Call<ComponentListKeys>;
```

Comment:
Is there a reason you don't do `export type $call = ...`?

Reply:
@AndrewSouthpaw 
Sorry i am not sure what it does. Not much experience in exporting type defs with flow.
i was trying to fix the default export issue by following https://github.com/facebook/flow/issues/1806#issuecomment-232194385
Also i can see some tests are failings. i'll have a look at it during the weekend.

Reply:
N/m, I was being derpy.

Looks like tests are still failing tho.

Reply:
@AndrewSouthpaw 
i wasn't exporting the $call and css with default and also some types
fixed now.
Tests pass on my machine. Node 9, the ci shows tests passing on Node 6, but fails on node 8.
Not sure why

Reply:
Tests can be flaky, just restarting the test fixed the issue.

Reply:
great . Thanks for merging